### PR TITLE
Add an admin role and move faculty to faculty.json

### PR DESCRIPTION
**Pull request by sommern**
Only 2 tests are working currently: `One Faculty` in `gkeepd_launch.robot` and `Add One Faculty` in `gkeep_add_faculty.robot`.

### CODE REVIEW

**Comment by sommern** [comment]
All tests are working now.

File: tests/acceptance/gkeep_add_faculty.robot [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Resource    resources/setup.robot
+Force Tags    gkeep_add_faculty
+
+*** Test Cases ***
+
+Add One Faculty
+    [Tags]    happy_path
+    Reset Server
+    Reset Client
+    Add File To Server    keeper    files/valid_server.cfg     server.cfg
```
Comment (bjcoleman) [inline tests/acceptance/gkeep_add_faculty.robot]: `Reset Server`, `Reset Client`, and `Add File To Server` are common to both tests and should be in a `Test Setup`.

File: tests/acceptance/gkeep_add_faculty.robot [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    Start gkeepd
+    Expect Email    to_user=admin_prof    contains=Password
```
Comment (bjcoleman) [inline tests/acceptance/gkeep_add_faculty.robot]: I think the check for an email to the admin is not appropriate here.  That functionality is tested in the `gkeepd_launch` tests.

File: tests/acceptance/gkeep_add_faculty.robot [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    Gkeep Add Faculty Succeeds    admin_prof    prof2
+    Expect Email    to_user=prof2    contains=Password
+    Server Running
+
+Duplicate Faculty
+    [Tags]    error
+    Reset Server
+    Reset Client
+    Add File To Server    keeper    files/valid_server.cfg     server.cfg
+    Start gkeepd
```
Comment (bjcoleman) [inline tests/acceptance/gkeep_add_faculty.robot]: Same comments as on the first test.

File: tests/acceptance/gkeepd_launch.robot [hunk 4]
```diff
@@ -24,71 +24,37 @@ Force Tags    gkeepd_launch
...
-    Expect Email    to_user=prof    contains=Password
-    Expect Email    to_user=prof2   contains=Password
-    User Exists    prof
-    User Exists    prof2
+    Expect Email    to_user=admin_prof    contains=Password
+    User Exists    admin_prof
     User Exists    tester
     Server Running
 
 Faculty Exists
```
Comment (bjcoleman) [inline tests/acceptance/gkeepd_launch.robot]: This should be `Admin Account Exists`

**Comment by sommern** [comment]
This pull request addresses issue #106

**Pull request merged**